internal/internal_decision_state_machine.go

```diff
@@ -454,16 +491,22 @@ func (d *commandStateMachineBase) String() string {
 
 func (d *activityCommandStateMachine) getCommand() *commandpb.Command {
 	switch d.state {
-	case commandStateCreated:
+	case commandStateCreated, commandStateCanceledBeforeSent:
 		command := createNewCommand(enumspb.COMMAND_TYPE_SCHEDULE_ACTIVITY_TASK)
 		command.Attributes = &commandpb.Command_ScheduleActivityTaskCommandAttributes{ScheduleActivityTaskCommandAttributes: d.attributes}
 		return command
 	case commandStateCanceledAfterInitiated:
-		command := createNewCommand(enumspb.COMMAND_TYPE_REQUEST_CANCEL_ACTIVITY_TASK)
-		command.Attributes = &commandpb.Command_RequestCancelActivityTaskCommandAttributes{RequestCancelActivityTaskCommandAttributes: &commandpb.RequestCancelActivityTaskCommandAttributes{
-			ScheduledEventId: d.scheduleID,
-		}}
-		return command
+		println("Act cmd get: ", d.id.String(), " sched: ", d.scheduleID)
```

Comment:
We shouldn't use println, use logger instead.

Reply:
These are all getting ripped out later. They're just in there until I fix the tests